test/governance/StkTruToken.test.ts

```diff
@@ -0,0 +1,175 @@
+import { expect, use } from 'chai'
+import { providers, Wallet } from 'ethers'
+import { solidity } from 'ethereum-waffle'
+
+import { setupDeploy } from 'scripts/utils'
+
+import { beforeEachWithFixture, DAY, parseEth, parseTRU, timeTravel } from 'utils'
+
+import {
+  MockOracle,
+  MockOracleFactory,
+  MockTrueCurrency,
+  MockTrueCurrencyFactory,
+  StkTruToken,
+  StkTruTokenFactory,
+  TrustToken,
+  TrustTokenFactory,
+} from 'contracts'
+
+use(solidity)
+
+describe('StkTruToken', () => {
+  let owner: Wallet
+  let staker: Wallet
+  let trustToken: TrustToken
+  let stkToken: StkTruToken
+  let tusd: MockTrueCurrency
+  let oracle: MockOracle
+  let provider: providers.JsonRpcProvider
+
+  const amount = parseTRU(100)
+  const stakeCooldown = DAY * 14
+
+  beforeEachWithFixture(async (wallets, _provider) => {
+    ([owner, staker] = wallets)
+    provider = _provider
+    const deployContract = setupDeploy(owner)
+    trustToken = await deployContract(TrustTokenFactory)
+    await trustToken.initialize()
+    tusd = await deployContract(MockTrueCurrencyFactory)
+    oracle = await deployContract(MockOracleFactory)
+
+    stkToken = await deployContract(StkTruTokenFactory)
+    await stkToken.initialize(trustToken.address, tusd.address, oracle.address)
+
+    await trustToken.mint(owner.address, amount)
+    await trustToken.approve(stkToken.address, amount)
+  })
+
+  describe('Oracle', () => {
+    it('only owner can change oracle', async () => {
+      await expect(stkToken.connect(staker).setOracle(staker.address)).to.be.revertedWith('only owner')
```

Comment:
```suggestion
      await expect(stkToken.connect(staker).setOracle(staker.address)).to.be.revertedWith('Ownable: Caller is not the owner')
```
Don't we want to have full revert messages in tests?

Reply:
These are some prehistoric contracts we don't want to change